acknowledgements.md

```diff
@@ -9,14 +9,14 @@ The development of the ESMValTool was/is funded by
 
 | Project | Description | Funding period |
 | ------- | ----------- | -------------- |
-| ![APPLICATE](/assets/img/APPLICATE_logo_square.png) | the EU's Horizon 2020 Advanced Prediction in Polar regions and beyond: Modelling, observing system design and LInkages associated with a Changing Arctic climate ([APPLICATE](https://applicate.eu/)) project" | -
-| ![BMBF](/assets/img/BMBF_Logo_square.png) | the BMBF's [CMIP6-DICAD](https://www.dkrz.de/c6de/dicad/) project funding CMIP6 activities in Germany" | - |
+| ![APPLICATE](/assets/img/APPLICATE_logo_square.png) | the EU's Horizon 2020 Advanced Prediction in Polar regions and beyond: Modelling, observing system design and LInkages associated with a Changing Arctic climate ([APPLICATE](https://applicate-h2020.eu/)) project" | -
+| ![BMBF](/assets/img/BMBF_Logo_square.png) | the BMBF's [CMIP6-DICAD](https://redmine.dkrz.de/projects/cmip6-dicad-subproject/) project funding CMIP6 activities in Germany" | - |
```

Comment:
I found two other pages which fit maybe better:
[1](https://c6de.dkrz.de/de-dicad.html) or [2](https://www.dkrz.de/de/projekte-und-partner/projekte/verbund-1-cmip6-dicad_tp1)
@axel-lauer You could have a look and decide but the one you choose was the internal project website for recording milestones.

With this change we could also add the funding period for CMIP6-DICAD: 07/2016-06/2020